axelor-gdpr/src/main/resources/domains/GDPRSearchConfig.xml

```diff
@@ -8,7 +8,7 @@
   <entity name="GDPRSearchConfig">
     <many-to-one ref="com.axelor.meta.db.MetaModel" name="metaModel" unique="true"/>
     <one-to-many ref="com.axelor.apps.gdpr.db.GDPRSearchConfigLine"
-      name="searchConfigLines" mappedBy="searchConfig"/>
+      name="searchConfigLines" mappedBy="searchConfig" title="GDPR Search configuration"/>
```

Comment:
```suggestion
      name="searchConfigLines" mappedBy="searchConfig" title="GDPR search configuration lines"/>
```

I'm not sure about my proposal, but I think we should distinguish the labels describing the object `GDPRSearchConfig` and the field `searchConfigLines`